.github/workflows/publish-pacts.yaml

```diff
@@ -4,12 +4,15 @@ on:
   workflow_dispatch:
   push:
     branches: [ dev ]
-
+env:
+  PUBLISH_CONTRACTS_RUN_NAME: 'publish-contracts-${{ github.event.repository.name }}-${{ github.run_id }}-${{ github.run_attempt }}'
+  CAN_I_DEPLOY_RUN_NAME: 'can-i-deploy-${{ github.event.repository.name }}-${{ github.run_id }}-${{ github.run_attempt }}'
 jobs:
   setup-and-test:
     runs-on: ubuntu-latest
     outputs:
       pact-b64: ${{ steps.encode.outputs.pact-b64 }}
+      new-tag: ${{ steps.tag.outputs.new_tag }}
```

Comment:
```suggestion
      new-tag: ${{ steps.tag.outputs.new_tag }}
      repo-branch: ${{ steps.extract-branch.outputs.repo-branch }}
```

Reply:
? There is no `tag` step in this workflow.